### DISCUSSION TITLE
[Feature] Request for Enhanced Album Sharing Features: Download and Screenshot Options

### DISCUSSION THREAD

**Discussion by AnjanModak** [immich-app/immich#12785] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be great to implement these two features, which all Imich users would definitely like:

1. When I share any album with others, I would like the option to choose whether to allow them to download photos from that album—yes or no—so that the person cannot download the photos.

2. Also, please disable the 'screenshot' and 'save as' options from the album.

This means that users would be able to see the photos but would not be able to download them or take screenshots.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
1. This is already a setting in the link sharing dialog
2. This isn't feasible. We need to get the image onto the user's device for them to look at, at which point we can't restrict them from keeping a copy anyways.
